distro/pom.xml

```diff
@@ -24,9 +24,9 @@
     <!-- track module versions
          we do so here so that we can utilise Maven to track updates, etc. -->
     <fhir2.version>2.0.0</fhir2.version>
-    <openmrs.version>2.6.6-SNAPSHOT</openmrs.version>
-    <initializer.version>2.7.0-SNAPSHOT</initializer.version>
-    <webservices.rest.version>2.44.0-SNAPSHOT</webservices.rest.version>
+    <openmrs.version>2.6.6</openmrs.version>
+    <initializer.version>2.7.0-20240521.045407-11</initializer.version>
```

Comment:
Can we get a non-SNAPSHOT release version of Iniz? Relying on SNAPSHOTs is causing us a lot of pain.

Reply:
Hi @ibacher!
What will be the version which should be used?

This is the latest Snapshot version after the last commit's release job

Reply:
<img width="459" alt="image" src="https://github.com/openmrs/openmrs-distro-referenceapplication/assets/51502471/dfecd38b-6683-48cb-8e44-3bc24687a0f5">

May I change the initializer version 2.6.0?

Reply:
I was thinking the same; it would be a mess to rebuild if those were cleaned up.

There are three other snapshots: Queue, Appointments, and Teleconsultation. I wonder why we reached that conclusion in the last release.